bin/create_yml.py

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env python
+
```

Comment:
can we get more info about this script?

Reply:
This script is for the differential RNA modification analysis using `xpore`, which requires a `config.yml` specifying the two sample conditions as input